### ISSUE TITLE
Change var webAppVersion = '6.0';

### ISSUE THREAD

**Issue by js-mike** [twa-dev/SDK#11]
Hi!,
Could you please update line 289 in telegram-web-apps.js var webAppVersion = '6.0'; from '6.0' to the latest version? The current 6.0 version crashes the app when used outside Telegram and use for example CloudStorage functionality.

**Comment by ArthurStam** [comment]
That's correct behaviour. 6.0 — is default value of this variable. TG client updates this var to it's real value when TMA starts . Value depends on TG client and its version.
